nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Product Search Page Complete

Fixes:
- Users can search product titles from the home page

Changes proposed in this request & reasons behind organizational or architectural decisions:
- Added a Search Results method and view to the Products Controller

Steps to test:
- Start Bangazon
- Enter the character "L" into the search bar
- Hit search

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-N/A

Commit message:
-search results bring up the search results view with products that match the entered search string
-changed search to search an all upper case normalized search string in an all upper ca se normalized product title"
--

Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-site-mighty-mahi-mahi/issues/8

@mighty-mahi-mahi
